dev/Materials/Acrylic/AcrylicBrush_rs2_themeresources.xaml

```diff
@@ -476,8 +485,16 @@
                 TintOpacity="0.8"
                 FallbackColor="#F3F3F3"
                 BackgroundSource="HostBackdrop"/>
```

Comment:
Can you update the non accent acrylic values to latest design POR in this PR or should I do it in a separate PR?

Here is the POR for light theme:
![image](https://user-images.githubusercontent.com/76921770/106033018-12aace00-6086-11eb-9357-04cfed143f73.png)